schema/tables/osquery_info.yml

```diff
@@ -1,4 +1,45 @@
 name: osquery_info
+columns:
+  - name: pid
+    platforms:
+      - macos
+      - windows
+      - linux
+  - name: uuid
+    platforms:
```

Comment:
Hey @eashaw, is it the case that these column attributes get merged with the osquery schema attributes as well? 

In other words, since I'm overwriting the column `uuid` with the platform information but not overwriting another attribute such as `description`, fleetdm.com/tables will show the original description, but with the new list of platforms?

Reply:
Not currently. We will need to update the script that generates pages for each table to support columns with three platform values. Right now, it would only add the first platform in the list if there are three platforms.

Reply:
ok got it. That's part of the specs for the work, when it comes around.

But that wasn't actually my question! Will it preserve the "description" column from the osquery schema?

Reply:
Sorry about that! Yes, it will.